corehq/apps/accounting/tasks.py

```diff
@@ -458,6 +460,27 @@ def auto_renew_subscription(subscription):
     )
     send_subscription_renewed_email(next_subscription)
 
+    if next_subscription.plan_version.plan.is_annual_plan:
+        invoice_contacts = [
+            subscription.account.billingcontactinfo.email_list
+            if BillingContactInfo.objects.filter(account=next_subscription.account).exists() else None
+        ]
```

Comment:
`email_list` looks like it would be an array, right? So this would set `invoice_contacts` to a 1-element or either the email list or `None`? Given the `not invoice_contacts` check right after, are the array brackets an error?
Nit, but it also seem more natural to use an empty array rather than `None` if no billing account exists.

Reply:
That's a good point. Resolved and moved to `get_contact_emails` on the prepayment invoice here https://github.com/dimagi/commcare-hq/pull/37094/commits/3db234d8b0932c2536d9119675e591ebbef98a4d